src/v2/view-builder/views/IdentifierView.js

```diff
@@ -36,6 +36,26 @@ const Footer = BaseFooter.extend({
 });
 
 export default BaseView.extend({
+  initialize () {
```

Comment:
Instead of being specific for `IdentifierView`, maybe we could pass the `messages` into View from FormController

```
this.formView = this.add(TheView, {
      options: {
        currentViewState,
        messages: this.settings.get(...),
      }
    }).last();
```

Then render the message from BaseView.